omero/sysadmins/cli/light-admins.rst

```diff
@@ -16,11 +16,13 @@ administrator restrictions in the awkward manner described hereunder.
 .. warning::
 
   OMERO.web provides a simplified view of the available restrictions:
-  checking *one* box in the web interface may lift *multiple* underlying
-  restrictions from an administrator. The recommended OMERO.web
-  management interface may thus prove confusing if OMERO.cli has been
-  used to set a combination of restrictions that does not correspond to
-  those bundles of related restrictions available in OMERO.web.
+  the :doc:`permissions mapping <../mapping-restricted-admins>` is such
+  that checking *one* box in the web interface may lift *multiple*
+  underlying restrictions from an administrator. The recommended
+  OMERO.web management interface may thus prove confusing if OMERO.cli
+  has been used to set a combination of restrictions that does not
```

Comment:
This certainly worries me. Is this something that's slated for 5.4.1? and/or should we consider a CLI plugin that we can rollout more quickly?

Reply:
Noted on https://trello.com/c/iZeXX5FD/17-manage-admin-privileges-from-cli though does require someone to more than just tweak existing CLI code.